src/Sylius/Behat/Page/Admin/Product/IndexPage.php

```diff
@@ -63,6 +73,7 @@ protected function getDefinedElements(): array
     {
         return array_merge(parent::getDefinedElements(), [
             'taxon_filter' => '.sylius-tree__item a:contains("%taxon%")',
+            'filter_channel' => '#criteria_channel',
```

Comment:
Keys should be in alphabetical order 😃 and to be consistent with an existing key, this one could be `channel_filter`